### PR TITLE
fix: remove emerge tools

### PR DESCRIPTION
**Pull request by anish-munirathinam**
# Description
removing emerge_tools.yml

## Links
n/a

# Checklist
- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [ ] I have added to the [changelog](../blob/master/CHANGELOG.md#Unreleased)
- [ ] I wrote/updated tests for new/changed code
- [ ] I removed all sensitive data **before every commit**, including API endpoints and keys, and internal links
- [ ] I ran `fastlane ci` without errors
- [ ] All project file changes are replicated in `SampleSPM/SampleSPM.xcodeproj` project

### CODE REVIEW

**Comment by maureenorea-clores** [comment]
Is it necessary to remove this file?

Is disabling workflow not enough?
https://github.com/rakutentech/ios-inappmessaging/actions/workflows/emerge_tools.yml

File: .github/workflows/emerge_tools.yml
Comment (Esakkiraja-Pothikannan) [inline .github/workflows/emerge_tools.yml:1]: For time being, Emerge workflow is disabled. This changes is not required.

**Comment by anish-munirathinam** [comment]
> Is it necessary to remove this file?
> 
> Is disabling workflow not enough? https://github.com/rakutentech/ios-inappmessaging/actions/workflows/emerge_tools.yml

Yes we should disable the workflow from GitHub Actions. I am simply testing it removing yaml file. Please ignore it, it automatically adds reviewers.

File: .github/workflows/emerge_tools.yml
Comment (anish-munirathinam) [reply .github/workflows/emerge_tools.yml:1]: Please ignore this, I am checking something. It automatically adds reviewer.

**Pull request closed without merging**